src/main.rs

```diff
@@ -1,5 +1,6 @@
 //! Main application entry point
 #![forbid(unsafe_code)]
+use std::borrow::Cow;
```

Comment:
🐄 ?

Reply:
Lol, it's a shorthand for `Clone on Write`.

Reply:
Love it!